docs/source/developing/writing-docs.adoc

```diff
@@ -24,71 +24,57 @@ submit a pull request with the changes.
 [[docs-conventions-guidelines]]
 == Conventions and guidelines
 
-Minishift documentation is authored in the AsciiDoc markup format. Since Minishift is closely related to the OpenShift Origin
+Minishift documentation is authored in the link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference[AsciiDoc] markup format. Since Minishift is closely related to the OpenShift Origin
 project, we follow the link:https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc[OpenShift Documentation Guidelines]
 for tagging, formatting, and structure wherever possible.
 
 The documentation source structure follows a modular format, where each file is called
 a _topic_ and each topic is stored under a sub-folder based on the relevant category, such as
 getting started, using Minishift, or command reference.
 
-TIP: You can check out the link:https://github.com/minishift/minishift/blob/master/docs/source/developing/writing-docs.adoc[raw file]
+TIP: You can check out the link:https://raw.githubusercontent.com/minishift/minishift/master/docs/source/developing/writing-docs.adoc[raw file]
 of this topic or other existing Minishift topics to see examples of how the information is structured and which tags to use.
 
 [[common-conventions]]
 === Commonly-used conventions
 
-- Each file must contain a topic metadata header with a top-level heading followed by required AsciiDoc variable declarations.
-The only exception to this is the command reference topics, which are automatically generated and do not require this header.
-You can view the raw file of this topic to see an example. Also, see
+- Each file must contain a topic metadata header with a top-level heading followed by required AsciiDoc variable declarations. See
 the link:https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#topic-metadata[topic metadata]
 section in the OpenShift documentation guidelines for more information.
 
-- Whenever you link to internal documentation topics, you should use relative links that can
-be resolved during the publication workflow instead of hard-coding URLs. See the link:https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#internal-cross-references[internal cross-references]
+- Links to another location within an AsciiDoc document or between AsciiDoc documents are called cross references and should use the _xref_ notation. See the link:https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#internal-cross-references[internal cross-references]
```

Comment:
I think that's what we decided on the meeting. To move this along the easiest is to align using xref.